crypto/s2n_stream_cipher_rc4.c

```diff
@@ -23,9 +23,13 @@
 
 static uint8_t s2n_stream_cipher_rc4_available()
 {
+#ifdef LIBCRYPTO_SUPPORTS_EVP_RC4
     return (EVP_rc4() ? 1 : 0);
+#endif /* LIBCRYPTO_SUPPORTS_EVP_RC4 */
+    return 0;
```

Comment:
None of the compilers get upset about two returns / unreachable code?

If might be cleaner if you use an #else

Reply:
Hmmm that is surprising. I'll make that change !